### ISSUE TITLE
The user cannot move items in Draggable list using keyboard anymore 

### ISSUE THREAD

**Issue by tiagodread** [ansible/awx#11467]
### Please confirm the following

- [X] I agree to follow this project's [code of conduct](http://docs.ansible.com/ansible/latest/community/code_of_conduct.html).
- [X] I have checked the [current issues](https://github.com/ansible/awx/issues) for duplicates.
- [X] I understand that AWX is open source software provided for free and that I am not entitled to status updates or other assurances.

### Summary

With the merge of https://github.com/ansible/awx/pull/11402 we lost the ability to move items in draggable list using only keyboard

### AWX version

devel

### Installation method

docker on linux

### Modifications

no

### Steps to reproduce

1. Go to organization add form
2. Open credential lookup
3. Try to reorder the credentials using only keyboard

### Expected results

The user should be able to do so using keyboard if he want's, look how this component used to work https://github.com/ansible/awx/pull/11402/files#diff-d5303e7af38edfc86cf59536805b488d5d5bf3d21f2b729bd6334023e7555cceL82-L91

### Actual results

Pressing enter or space to start the move then pressing arrow up/down is not moving the item

**Comment by nixocio** [comment]
See: https://github.com/ansible/awx/pull/11402#pullrequestreview-829224463

 https://www.patternfly.org/v4/components/drag-and-drop

> Note: Keyboard accessibility and screen reader accessibility are still in development.

**Comment by tiagodread** [comment]
The test case `Can create an org and reorg the credential order on edit` is failing because cypress can't reorder the items using keyboard anymore

**Comment by AlexSCorey** [comment]
This is currently under development with PF https://github.com/patternfly/patternfly-react/issues/6317

**Comment by AlexSCorey** [comment]
PF has decided the deprecate the BETA component I've used in this [PR](https://github.com/ansible/awx/pull/11402/files) so we may want to revert it, which I think will resolve this issue.

**Comment by tiagodread** [comment]
> 

@AlexSCorey make sure all feedback Kersom reported in the original PR is working properly in the revert PR.

**Comment by akus062381** [comment]
resolved
